src/main/java/com/rakovets/course/java/core/practice/operators/Task05.java

```diff
@@ -36,6 +36,6 @@ public static void main(String[] args) {
     static int calculateDeviationStandardWeight(int height, int weight) {
         //TODO
         // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return 0;
+        return weight-(height-110);
```

Comment:
"Violation: There isn't separating white spaces for operators OR parameters
Solution: Add white spaces"